src/Presence/apple.json

```
@@ -0,0 +1,282 @@
+        "message": "Plan Your Visit",
+        "description": "Plan your visit to the Apple Store"
+    },
+    "apple.shopRefurbished": {
+        "message": "Certified Refurbished",
+        "description": "Used Apple products at a lower price"
+    },
+    "apple.shopBag": {
+        "message": "Shop - Bag",
+        "description": "Shopping cart"
+    },
+    "apple.shopBagSingle": {
+        "message": "Bag",
+        "description": "Shopping cart"
+    },
+    "apple.shopBagSummary": {
+        "message": "Summary: %summary%",
+        "description": "Your total amount of the bag(shopping cart)"
+    },
+    "apple.support": {
+        "message": "Apple - Support",
+        "description": ""
+    },
+    "apple.supportTwo": {
+        "message": "Apple Support",
+        "description": ""
+    },
+    "apple.supportArticle": {
+        "message": "Apple Support - Article:",
+        "description": "Support article"
+    },
+    "apple.appStoreCon": {
+        "message": "Connecting...",
+        "description": "Connecting to the App Store"
+    },
+    "apple.iCloudMail": {
+        "message": "Mail",
+        "description": "iCloud mail service"
+    },
+    "apple.iCloudContacts": {
+        "message": "Contacts",
+        "description": "iCloud contacts"
+    },
+    "apple.iCloudCalendar": {
+        "message": "Calendar",
+        "description": "iCloud calendar"
+    },
+    "apple.iCloudPhotos": {
+        "message": "Photos",
+        "description": "iCloud photos"
+    },
+    "apple.iCloudNotes": {
+        "message": "Notes",
+        "description": "iCloud notes"
+    },
+    "apple.iCloudReminders": {
+        "message": "Reminders",
+        "description": "iCloud reminders"
+    },
+    "apple.iCloudPagesCreate": {
+        "message": "Creating...",
+        "description": "Creating a new Pages document"
+    },
+    "apple.iCloudKeynoteWait": {
+        "message": "Waiting room...",
+        "description": ""
+    },
+    "apple.devNew": {
+        "message": "What's new",
+        "description": "New things"
+    },
+    "apple.devSubmit": {
+        "message": "Submissions",
+        "description": "Submit iOS and iPadOS apps to the App Store"
+    },
+    "apple.devPhonePad": {
+        "message": "iPhone and iPad apps",
+        "description": ""
+    },
+    "apple.devDistribution": {
+        "message": "Distributing software",
+        "description": ""
+    },
+    "apple.devDocs": {
+        "message": "Documentation",
+        "description": "Developer documentation"
+    },
+    "apple.devHIG": {
+        "message": "Human Interface Guidelines",
+        "description": ""
+    },
+    "apple.devResources": {
+        "message": "Resources",
+        "description": "Apple Design Resources"
+    },
+    "apple.devFeatures": {
+        "message": "Features",
+        "description": ""
+    },
+    "apple.devGames": {
+        "message": "Games",
+        "description": "Games like Minecraft etc."
```

I believe this is what you mean
```suggestion
        "description": "Apps classified under the \"Games\" category on the App Store"
```